plugins/SegmentEditor/API.php

```diff
@@ -107,16 +108,11 @@ protected function checkAutoArchive($autoArchive, $idSite)
         }
 
         if ($autoArchive) {
-            if ($realTimeSegmentsEnabled && !Piwik::isUserHasAdminAccess($idSite)) {
-                // pre-processed segment for a given website requires admin access
-                throw new Exception(
-                    "Please contact Support to make these changes on your behalf. ".
-                    " To modify a pre-processed segment, a user must have admin access or super user access. "
-                );
-            } elseif (!$realTimeSegmentsEnabled) {
-                // we require view access only when only pre processed can be created
-                Piwik::checkUserHasViewAccess($idSite);
+            if (Rules::isBrowserTriggerEnabled()) {
+                throw new Exception("Pre-processed segments can only be created if browser triggered archiving is disabled.");
```

Comment:
Could we add a link to https://matomo.org/docs/setup-auto-archiving/  if it's a super user? Like `To disable browser archiving follow the instructions in https://matomo.org/docs/setup-auto-archiving/ .` Just to give them an idea of what to do next.